arcgis-ios-sdk-samples/Content Display Logic/Storyboard/LaunchScreen.storyboard

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thank you for converting the launch screen nib into a launch storyboard. With a storyboard, we can more closely model the main view of the app on app launch. I spent quite a bit of time playing with the colors to get them right, but there just doesn't seem to be a way. Very frustrating. In any case, please make the following changes:

- Mark the storyboard as a launch storyboard.
  ![Screen Shot 2020-09-02 at 7 12 48 AM](https://user-images.githubusercontent.com/2257493/91994742-cb452f00-eceb-11ea-83b0-7b073a841ef0.png)
- Use a split view controller. I know we can't get the colors right, but we can at least get the other aspects of the launch image correct. The goal is for the launch image to resemble the initial view of the app as closely as possible.

Reply:
The 2 issues you mentioned are fixed. Now the LaunchScreen uses a `UISplitViewController`. There are a few comments and issues:

![demo](https://user-images.githubusercontent.com/9660181/92157044-b8f7ed80-edde-11ea-9d84-5449a2b8de73.gif)

- On iOS 12, the LaunchScreen is free of issue 🎉  for
    - Landscape: iPad, iPhone
    - Portrait: iPhone
- For iOS 12 iPad Portrait mode, only the detail VC is displayed in LaunchScreen. A hack to solve it is [here](https://stackoverflow.com/questions/38564870/uisplitviewcontroller-allvisible-in-launchscreen-storyboard).
- On iOS 13, due to the new "prefer large title" attribute for navigation bar, a blank screen will show for both iPhone or iPad. 
    - I tried several Apple native apps with navigation bar on their first screen (Apple Research, TestFlight, Health) and none of them show a "placeholder frame" of the nav bar with large title. So I assume showing a blank screen is actually the expected behavior for iOS 13 and above.
- On iOS 13 and iOS 14 the searchBar is displayed, while on iOS 12 it is hidden. It is probably due to [this](https://stackoverflow.com/questions/57581557/how-to-initally-hide-searchbar-in-navigation-controller-on-ios-13). I wonder if we should make it consistent as iOS 12's behavior on iOS 13 and above?

Reply:
Thanks for doing all that research! What you have is fine.

Reply:
Thanks. I'll add Sarat to review this and also roll out another TestFlight version. After that I'll post in cocoa channel to let  our team give it a try.